sentry_plugin/hooks/sentry_hook.py

```diff
@@ -70,27 +93,20 @@ class SentryHook(BaseHook):
     """
 
     def __init__(self, sentry_conn_id=None):
-        integrations = []
         ignore_logger("airflow.task")
+        ignore_logger("airflow.jobs.backfill_job.BackfillJob")
         executor_name = configuration.conf.get("core", "EXECUTOR")
 
         sentry_flask = FlaskIntegration()
+        integrations = [sentry_flask]
 
         if executor_name == "CeleryExecutor":
             from sentry_sdk.integrations.celery import CeleryIntegration
+            from sentry_sdk.integrations.tornado import TornadoIntegration
 
             sentry_celery = CeleryIntegration()
-            integrations = [sentry_celery]
-        else:
-            import logging
-            from sentry_sdk.integrations.logging import LoggingIntegration
-
-            sentry_logging = LoggingIntegration(
-                level=logging.INFO, event_level=logging.ERROR
-            )
-            integrations = [sentry_logging]
-
-        integrations.append(sentry_flask)
+            sentry_tornado = TornadoIntegration()
```

Comment:
Do you want to add this to the integrations list below?

Reply:
The tornado has been causing some issues when I test celery on Google Composer. This is one reason why I haven not been able to merge this PR just yet. I will get this resolved by tomorrow latest.

Reply:
Curious what issues you've been running into? I've been running this on Heroku and things seem to be working without any major issues.

Reply:
Tornado requires Python3.6+ or the aiocontextvars package. For some reason this is creating an issue with using the `sentry_dsn` from the connections database, and the exceptions are not sent to Sentry.